.travis.yml

```diff
@@ -18,14 +18,12 @@ language: java
 
 services:
   - docker
-  
+
 jdk:
   - oraclejdk8
-install: mvn clean install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
`mvn dependency:resolve` actually fails here. I'll put a noop for install for now.